File: DemoServer/AdditionalLanguages/python/relatedDocuments/loadRelatedDocuments/loadRelatedDocuments.py
from demo_example import Example, RunParamsBase
from models import Product, Supplier

class RunParams(RunParamsBase):
    def __init__(self, price_per_unit: float, phone: str):
        self.price_per_unit = price_per_unit
        self.phone = phone

class LoadRelatedDocuments(Example):
    def run(self, run_params: RunParams) -> None:
        pricePerUnit = run_params.price_per_unit
        phone = run_params.phone

        #region Demo
        with self.document_store_holder.store().open_session() as session:
            #region Step_1
            product = session\
                .include("Supplier")\
                .load(Product, "products/34-A")["products/34-A"]
            #endregion

            #region Step_2
            supplier = session.load(product.Supplier, Supplier)
            #endregion

            #region Step_3
            product.PricePerUnit = pricePerUnit
            supplier.Phone = phone
            #endregion

            #region Step_4
            session.save_changes()
            #endregion
        #endregion

File: DemoServer/AdditionalLanguages/php/relatedDocuments/loadRelatedDocuments/LoadRelatedDocuments.php
<?php

namespace RavenDB\Demo\relatedDocuments\loadRelatedDocuments;

use RavenDB\Demo\common\DocumentStoreHolder;
use RavenDB\Demo\common\models\Product;
use RavenDB\Demo\common\models\Supplier;

class LoadRelatedDocuments
{
    public function __invoke(RunParams $runParams): string
    {
        $pricePerUnit = $runParams->getPricePerUnit();
        $phone = $runParams->getPhone();

        //region Demo
        $session = DocumentStoreHolder::getStore()->openSession();
        try {
            //region Step_1
            $product = $session
                ->include("Supplier")
                ->load(Product::class, "products/34-A");
            //endregion

            //region Step_2
            $supplier = $session->load(Supplier::class, $product->getSupplier());
            //endregion

            //region Step_3
            $product->setPricePerUnit($pricePerUnit);
            $supplier->setPhone($phone);
            //endregion

            //region Step_4
            $session->saveChanges();
            //endregion
        } finally {
            $session->close();
        }
        //endregion

        return "Document $documentId was edited successfully";
    }
}
